### DISCUSSION TITLE
Commissioning Issue In QPG6105 Matter - Setup

### DISCUSSION THREAD

**Discussion by blessendanielraj** [openthread/openthread#9748] [Q&A]
Hi Guys
Just recently we started settingup up matter environment using below setup

a. QPG6105 with Matter light application
b. OpenThread Border-router QPG7015M
c. POSIX-CLI chip tool
d. Environment: Ubuntu 22

We are facing issue in the commissioning part of the setup and not sure if it’s issue with the matter node (QPG6105) or with the Border router.

Below i am attaching the screenshot of the QPG6105 matter node and also the progress we’ve done so far in the format of excel. Please give us some insights on setting up qorvo border router.

Thanks in advanced.
![Commissioning Issue in Qorvo](https://github.com/openthread/openthread/assets/89910804/4fe702c0-3610-4744-a8fe-389f162cfffb)

**Comment by jwhui** [comment]
Can you attach the full log in a text file?
